stacks_queues/stack.py

```diff
@@ -21,18 +21,30 @@ def pop(self):
             The Stack is empty.
             returns None
         """
-        pass
+        if self.store == None:
+            raise StackEmptyException("Stack is empty!")
+
+        removed = self.store.get_first()    
+        self.store.remove_first()
+
+        return removed
+
 
     def empty(self):
         """ Returns True if the Stack is empty
             And False otherwise
         """
-        pass
+        if self.store.get_first() == None:
+            return True
+    
+        return False
 
     def __str__(self):
```

Comment:
👀 Again, take a look at the LinkedList implementation! There's a `__str__` method there for you to use.